SwiftVoiceQuickstart/ViewController.swift

```diff
@@ -109,10 +109,13 @@ class ViewController: UIViewController, PKPushRegistryDelegate, TVONotificationD
     }
 
     @IBAction func placeCall(_ sender: UIButton) {
-        if (self.call != nil && self.call?.state == .connected) {
-            self.userInitiatedDisconnect = true
-            performEndCallAction(uuid: self.call!.uuid)
-            self.toggleUIState(isEnabled: false, showCallControl: false)
+        if (!self.activeCalls!.isEmpty) {
+            let call = self.activeCalls![0]
+            if (call.state == .connected) {
```

Comment:
Ideally, we should follow the native dialer app rule - when a user makes an outbound call, the existing call goes on call hold state. But since we update the app UI of the sample app, for this PR, I guess we can keep the simple rule of disconnecting the existing call.

As discussed, change we need to make here is to disconnect the call regardless of the state of the Call.